.NET/Microsoft.Recognizers.Text.DateTime/Parsers/DummyTimeZoneParser.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    public class DummyTimeZoneParser : IDateTimeParser
+    {
+
+        public ParseResult Parse(ExtractResult result)
+        {
+            return Parse(result, DateObject.Now);
+        }
+
+        public List<DateTimeParseResult> FilterResults(string query, List<DateTimeParseResult> candidateResults)
+        {
+            return candidateResults;
+        }
+
+        public DateTimeParseResult Parse(ExtractResult er, DateObject refDate)
+        {
+            return null;
```

Comment:
If there is no output from Parse, should it really be null or an empty DateTimeParseResult?